### ISSUE TITLE
Failed to format file because java.lang.NullPointerException

### ISSUE THREAD

**Issue by yenda** [kkinnear/zprint#230]
The following minimal repro

```clojure
(ns test
  (:require [clojure.spec.alpha :as s]))

(s/cat)
```

with the following:
```shell
$ clojure -A:zprint '{:cwd-zprintrc? true}' -lsc ./test.clj
WARNING: When invoking clojure.main, use -M
Processing file ./test.clj
Failed to format file: ./test.clj because java.lang.NullPointerException
Processed 1 file, with 1 error, none of which requires formatting.
```

**Comment by kkinnear** [comment]
Thans for filing an issue!  Unfortunately, I can't reproduce this, despite the simple test case.  I'm guessing that something in your `.zprintrc` file is interacting with the source in some way.

If you could let me know what version of zprint you are using:
```
% clojure -A:zprint --version
```
and give me the contents of your `~/.zprintrc` or `~/.zprint.edn` file and the local `.zprintrc` or `.zprint.edn` file, I expect that I'll be able to reproduce this.

Thanks again for submitting an issue.

**Comment by yenda** [comment]
My bad I forgot about zprintrc

```
{:fn-map { "s/cat"         [:binding {:binding {:force-nl? false}}]}
 }
```

I'm using 1.2.2

**Comment by kkinnear** [comment]
Thanks!  Yes, the `:binding` function type has a problem when there is only one element in the list.  I have fixed this, and the fix will be in version `1.2.3`, which is coming out soon.  I was just starting the release process, so this one just made it in.

Thanks again for reporting this!
